### ISSUE TITLE
Different icon for benches with backrest

### ISSUE THREAD

**Issue by Tomasz-W** [gravitystorm/openstreetmap-carto#3187]
Due to Taginfo, out of 872k benches, 294k of them are tagged with backrest=yes.
https://taginfo.openstreetmap.org/tags/amenity=bench#overview

As sitting on non-backrest benches is not comfortable, I think there should be 2 different icons for 2 different bench types.

* ![pobrany plik](https://user-images.githubusercontent.com/25656654/39094321-f59df91a-462d-11e8-82a8-c14284494991.png) (current amenity=bench icon)
* ![amenity bench backrest yes](https://user-images.githubusercontent.com/25656654/39094324-096fe516-462e-11e8-85c9-0f87f7099477.png) (amenity=bench + backrest=yes icon proposal)

**Comment by matkoniecz** [comment]
I am strongly against this idea. It is not important info, it is not relevant for orientation, it is too detailed for a general purpose map.

**Comment by eehpcm** [comment]
Sounds good to me.

**Comment by IVAN-RSM** [comment]
Nice

**Comment by matkoniecz** [comment]
See https://github.com/gravitystorm/openstreetmap-carto/blob/master/CARTOGRAPHY.md#main-goals

> The aim is not however to show all or even most of the OSM data.

Unfortunately showing everything is not feasible.

**Comment by eehpcm** [comment]
> Unfortunately showing everything is not feasible.

Indeed.  Last week I mapped a public space which had two picnic tables.  Sadly, they are too close together and so only one is shown by the renderer.  I can live with that.  I understand.  There's not enough room to fit in two symbols that close together.  I can wish for higher zoom levels so they can be shown, but I don't expect that to happen (my wish for lots of money didn't come true, either, but 
that doesn't stop me wishing).

This is different.  Icons are standard sizes and I think the proposed bench-with-back icon fit into that standard size.  If I'm wrong and it was a little larger, it could be redrawn to fit.  So if it's feasible to fit a bench-without-back icon it's feasible to fit a bench-with-back icon.  The hard part (once a symbol that fits the standard icon size has been created) is modifying the carto code to choose one symbol or the other depending on whether or not it has backrest=yes.

I can understand "it's too much work when we have so many other issues to deal with" but not "showing everything is not feasible" when in this case showing it is feasible.  Nor can I understand "it's too much detail" when, for some people, it may be an important consideration.

Just my opinion.  You have the final say.

**Comment by kocio-pl** [comment]
It's interesting to me to hear what it means "too much". As long as we can reasonably easy (design is simple, it's not adding clutter and not eating our resources) show something very popular, we should do it IMO.

I believe vector styles will help to choose the lighter or richer style, and even personalize it, and I hope this will make such discussions moot eventually.

**Comment by matkoniecz** [comment]
> Compared to 876 820 amenity=bench, almost every second bench has this attribute.

Are you sure that it is not result of a large import?

Also, I mapped some backrests with StreetComplete, it does not mean that I think that this data should appear on a general purpose map.

**Comment by kocio-pl** [comment]
> See this graph from taghistory.raifer.tech:

The graph is missing.

**Comment by Tomasz-W** [comment]
@matkoniecz As there are people people interested in this feature and we don't have test renderings yet, please re-open this issue. Closing it that early was unfair to another users.

**Comment by kocio-pl** [comment]
OK, but reopening won't change too much, the most important thing is having discussion and new arguments or decisions.

**Comment by eehpcm** [comment]
@kocio-pl 

> the most important thing is having discussion and new arguments

OK.  Not "all-new" but some new and some rehashing...

1. For some of us, this is an important detail.  If you have back problems, the presence or absence of a back rest determines whether or not the bench is usable.  Perhaps it's more psychological than physical (at least in my case) but a bench without a backrest gives me back pain after a few minutes.

2. The clutter issue doesn't apply.  If there's room to render a bench without a rest there's room to render a bench with a rest.  An area dense with detail may prevent a bench being rendered at all, but if a bench can be rendered a bench with a backrest can be rendered (assuming one can be drawn at the same size of icon).

3. As I see it (from my Dunning-Kruger understanding) one possible issue is that adding extra code to handle the two cases makes the rendering process hit a brick wall.  As I understand it, even minor additions of conditionals can result in major increases in rendering time (by that I mean all stages involved in transforming the raw data into tiles).  We won't know for sure unless somebody tries it.

4. The only other major possible problem is that there is too large a backlog of more important things to do.  But importance is determined, in part, by how much demand there is for a particular feature.

5) One can spend some time adding a bench with a backrest or one can spend some time defending why benches with backrests will not be added every time the issue crops up.

**Comment by Tomasz-W** [comment]
Pixel aligned icon proposals:

1) ![amenity bench backrest yes](https://user-images.githubusercontent.com/25656654/44873403-6f78e300-ac98-11e8-95ca-1f20810b373e.png)
2) ![amenity bench backrest yes](https://user-images.githubusercontent.com/25656654/44873482-a818bc80-ac98-11e8-90c6-a94c752f72e9.png)

Gist link: https://gist.github.com/Tomasz-W/5703292edb2d8b09aab74291636faaf2

If someone is going to make a test renderings, please remember about https://github.com/gravitystorm/openstreetmap-carto/issues/3326

**Comment by sommerluk** [comment]
I share the concerns of @matkoniecz 

Further concerns are:

1. The proposed icon is not monochromatic. If full colours are used, this is problematic for a consistent colour usage (and also for automatic colour styling in mss files), and transparencies can be problematic also.

2. The problem is maybe not “too much clutter”, but over-differentiated icons. When we have too many different icons (that are not really very very intuitive), sooner or later people, especially casual users, while fail to recognize the meaning of individual icons. While rendering `backrest=no` would be in line with the main goal “A rich map”, it would be not in line with the main goal “Legibility and clarity”. I do not think the gain is worth the cost.

3. What makes sure that `backrest=no` is a sensible default when this tag is missing?

**Comment by eehpcm** [comment]
@sommerluk 

> When we have too many different icons (that are not really very very intuitive), sooner or later people, especially casual users, while fail to recognize the meaning of individual icons.

I agree that some of the icons aren't intuitive.  But is it really better to use a small disc instead?  For **everything**?  Surely not.  So who decides "this gets an icon but that doesn't"?  Because somebody has to draw the line if we decide not to provide icons for everything.  Which ones get an icon and which don't and why?  I don't use certain types of facility so I see no need for them to have icons: I've never bought a bicycle so the icon for bike shops is a waste of map space as far as I'm concerned.  So which person decides "this is useful and should appear on a map but that isn't useful" and can do so **objectively**?

It's possible that two or more instances of a non-obvious icon will be present on the map, one with text rendered making it clear what it is and one without text.  Users can at least use that to infer what the unlabelled icon is.  Whereas if they both have small purple discs there is no clue as to what the unlabelled one is.

Your argument indicates that it's time the map legend included the icons as well as the cartography used for roads.  Possibly on a separate button so there are two side panels available, the current one and a map symbols panel.

**Comment by sommerluk** [comment]
While I agree that bicycle shops are not a high priority, the icon is quite intuitive, will not be confused with other things, and the purple colour makes clear immediately that this is a shop. That’s different from the proposed bench icon.

> it's time the map legend included the icons as well as the cartography used for roads

At this place I want to quote our guidelines at CARTOGRAPHY.md:

> The map should be intuitively readable by users with some general experience using maps without a map key, preferrably with relatively little effort.

**Comment by kocio-pl** [comment]
This addition is subtle and it is very similar to the bench icon, so I expect no problems.

**Comment by eehpcm** [comment]
@summerluk

I'll deal with your points out of order.

> At this place I want to quote our guidelines at CARTOGRAPHY.md:

> >  The map should be intuitively readable by users with some general experience using maps without a map key, preferrably with relatively little effort.

I agree 100%.  My opinion is that map keys and FAQs about user interfaces mean the map carto or UI is sub-optimal.  UIs should have infrequently-asked questions to cover very rare use cases, the rest should be self-explanatory, or discoverable with tool tips.  But we live in an imperfect world.  Not all of the map icons are intuitive.  Maybe some (perhaps even most) of them could be improved.  But a pane with map symbols would be there as a last resort.  Because with a poor icon and a pane of map symbols I can figure out what something is, but with nothing more than a small disc I cannot do more than place it into a broad category of shop or amenity or some such.

> While I agree that bicycle shops are not a high priority, the icon is quite intuitive, will not be confused with other things, and the purple colour makes clear immediately that this is a shop. That’s different from the proposed bench icon.

Of all the things on the map I don't use, and am never likely to use, I chose bike shops because I'm well aware that OSM has a high level of input from the hiking/biking/touring community.  We should strive to cast aside our own subjective preferences when judging which facilities are useful enough to deserve an icon.  Otherwise I **demand** bike shops no longer have an icon. :)

And yes, the purple bike icon is obvious and it's clear it's a bike shop.  How obvious would it be if I had my way and it were demoted to a small purple disc?  Unless the name of the shop has "bike" or "bicycle" in the name _and_ the map is uncluttered enough at that location for the name to render, it wouldn't be obvious.

At least we've progressed from "I don't use this so there's no need to map it as anything other than a disc." to "I find it difficult to figure out what this icon is so lets have a disc even if it's obvious to other people what the icon means."

To me it still seems that the problem is that we need better icons and that until we have better icons a map symbols pane would mitigate the problem.  In the longer term vector mapping may mean we have fancy JSON trickery so you hover over an object and a pane pops up giving more details of anonymous discs.  But when we have vector mapping we'll be able to zoom in so much we can use larger, clearer icons at the highest zooms.

**Comment by Adamant36** [comment]
Personally I dont see how its not obvious what @Tomasz-W's icon is. Theres really nothing else it can be mistaken for that I can think of. So I dont see whats unintuitive about it. If there is a slight chance someone wont get it automatically though the context around it like woods etc would probably make it clear.

**Comment by polarbearing** [comment]
Which woods?

So we would need at least 3 icons, one for yes, one for no, and one for the ca 50% untagged?

**Comment by Adamant36** [comment]
I dont know. Its called an example. Pick a random bench in the woods. The point was it probably wont be confused for a store icon or anything and its the places where they are located usually helps. Like, theres only so many icons that would be in a park along a path for instince. People use the surrounding context in identifying things on a map to some degree even if the icon is perfectly clear. The bench icon clearly looks like a bench though. So I dont know why it even matters. At least to me. Otherwise, what else does it look like or how specifically is it unintuitive?

**Comment by meased** [comment]
The bench+backrest icon does not look like a bench to me.

Going by @kocio-pl's size argument, a bench is already a very small object, and now we're talking about rendering a detail on a small object? Playground equipment is much bigger and more important that the back of a bench, and that was rejected. I guess we should also rethink grit bins, light poles, and surveillance cameras too.

**Comment by kocio-pl** [comment]
The difference is that benches have well known shape and playground equipment icons does not exist yet (I'm taking about real 14 px version), so they had no chance to be rendered.

But playgrounds have their icon, even if they are twice as less popular than benches:
![taghistory 34](https://user-images.githubusercontent.com/5439713/44885066-32741700-acbf-11e8-88a5-8d4492fa6dfb.png)

The back rest is very popular, so we would promote more precise tagging with rendering small additional detail for very popular tag:
![taghistory 35](https://user-images.githubusercontent.com/5439713/44885208-dbbb0d00-acbf-11e8-9032-4fdd7d218eb9.png)

In fact back rest benches are comparable in number with playgrounds, so it makes sense to show them a bit differently. It's also easy to notice if they have back rest or not.

**Comment by polarbearing** [comment]
So where lies the backrest=no in the graph, and what to do with the untagged?

**Comment by Tomasz-W** [comment]
@polarbearing proposed icon is only for combination with `barkrest=yes`, all the rest would stay with current generic icon

**Comment by kocio-pl** [comment]
The usual way is to make generic tags look plain and simple. Simple bench does not promote adding backrest=no, because they will still look the same, however at least we would promote adding the feature, because there is a visible change.

**Comment by imagico** [comment]
Traditionally this style always aimed not to assume defaults for supplemental tags unless they are generally accepted by the community (like access tags) and have a basis in reality (i.e. for the vast majority of features the default does apply) because this would clash with the goal on providing constructive and intuitive feedback to mappers.  Therefore we do not render forests without leaf_type like mixed forests but indicate the lack of leaf_type tagging explicitly in styling.

**Comment by polarbearing** [comment]
I'm not against rendering backrest=* (though it's not on my priorities, but I accept it is some mappers playground). However when doing so both values should be addressed. Today's stats:
amenity=bench 943368
backrest=* 388831
backrest=no 66405
backrest=yes 322222

**Comment by matthijsmelissen** [comment]
This is really getting to detailed. Increasing the number of icons used on the map will make it harder to recognize/learn the meaning of icons. This has also already been stated by @matkoniecz and @SomeoneElseOSM, so I'm going to close this issue again.

**Comment by eehpcm** [comment]
@matthijsmelissen 

> This is really getting to detailed. Increasing the number of icons used on the map will make it harder to recognize/learn the meaning of icons

Do you have a citation to back that up?  Have formal studies been done?  Or are you merely reasoning that more icons means more to recognize and leaping to the conclusion that more icons make the map less usable?  I don't think it's that simple.

If **all** icons are replaced by the appropriately-coloured disc then nothing is recognizable yet we've "simplified" things greatly.  So it's clearly more complex than "fewer is better."  I repeat the questions I asked earlier: if we restrict which objects get icons then who decides and how do we ensure it is done objectively?

Here are two ways in which a bad icon (bad meaning unrecognizable, not misleading) is better than the anonymous coloured disc:

1. I at least know what it is **not**.  It it not the same class of object as that _different_ icon of the same colour I can't decipher.  With discs all I know is that I don't know what they are.  They might be the same class of object or they might not.  With icons, even if one looks like a fly on its back and the other looks like a UFO, I know they're not the same class of object.  Not a great improvement, but an improvement.

2.  I know when it **is** the same class of object as another that is in view or I have seen recently and remember.  One of them might have its name rendered, which allows me to deduce what type of object it is.  For example, the icon for a butcher isn't very obvious to me, but if I see that icon with the label "Keith's Butcher" then I know that the other place I see that icon is also a butcher.  If neither is labelled and I use the query tool to find out that one is a butcher I don't have to repeat the query to find out what the other one is.  With anonymous discs I have no choice but to query both.  And maybe a lot of other anonymous discs with no labels.

As a general argument, I find the idea that more icons will make the map harder to use is unconvincing.  In this particular case, benches, it's hard to imagine what other kind of sidewalk object they could be confused with.

**Comment by Adamant36** [comment]
@matthijsmelissen, I don't think SomeoneElseOSM is a good example here. His map is more detail in some respects then ours. For instance he renders street lamps when we don't at the moment. So your really picking and choosing. I also think its a little out there to close this issue based on your own opinion on it when there is clearly support for it and the whole "there is no consensus anymore" discussion is going on. Like @eehpcm has said, where is the evidence to back up your assertions and why is it applicable to this particular icon? The important thing is that it acts as a feedback mechanism for more people add the backrest tag. Which I think it will do. 

You can't really use the "clutter" or "to many icons" argument here because bench's are already being render. The bottom half of the bench with the back rest is exactly like the normal bench. Your making a major assumption about the intelligence of mappers if you believe they wont make the connection between the two in this case. Otherwise, back it up with some tests or something. Even @polarbearing said he wasn't against it and he's usually pretty bearish on things not being rendered excessively. That should tell you something.

**Comment by Adamant36** [comment]
@sommerluk, notice I said didn't say _majority_ or even _equal_. There has been a few PRs that I was involved in where the majority where in favor of it but only @polarbearing was against it. Me and the others were perfectly willing to not take action on it until he had been heard out though. Although I know discussion is necessarily killed off by an issue being closed, I still don't see why it is necessary here. Even if three maintainers are against it. There are plenty of open issues going years back that had zero favor but never got closed. There's no reason this one should be unique. Especially considering the discussion was still on going. He could of at least waited until it petered out or the test renders ended up failing massively. Then at least there could have been a reasoned argument for closing it. Where does it say that maintainers have ultimate say over the rest of community anyway? 

I'll also point out the issue for rendering a cross which was closed at the same time by @matthijsmelissen for the same none reason, when it could have just as easily stayed open also or he could of at least asked what other's thought of closing it. Taking both the issues together at the some time like they where makes me think it was more reactionary then anything. Otherwise, why not indulge me, @eehpcm, and the other people for it by rolling the dice and providing evidence for it shouldn't be rendered. Using the low hanging fruit as it where as reasons makes me think its more cartographic dogma then anything well reasoned though. Which is fine. Personally, id like to see it reopened and at least tested though. If the test turn out bad. Fine, but I why kill it off hand before that point?

**Comment by Tomasz-W** [comment]
Let's count. Basing on comments/ likes:

* people who supported the idea or said that they are not against: @Tomasz-W @eehpcm @IVAN-RSM @dieterdreist @kocio-pl @Adamant36 @polarbearing [7]
* people against: @matkoniecz @igoreliezer @geozeisig @jbelien @meased @sommerluk [6]

I didn't count Imagico's comment, because it looks more like an information than a statement in this particular issue.

We didn't see even single one test rendering, and with around 50:50 "votes" @matthijsmelissen closed the issue. I think after this situation he shouldn't have the power to closing them, or at least get some warning as he showed big disrespect for another users. There is no point in our Terms of use (https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md) that maintainers votes are more important than other users opinion.

**Comment by matkoniecz** [comment]
> with around 50:50 "votes"

Decisions should be based on arguments, not on votes. I admit that sometimes it is really hard to judge arguments, but it is not a popularity contest, and just "X people want Y" is never going to be a deciding argument.

**Comment by matkoniecz** [comment]
> there were no reason to closing the ticket

There were reasons given above both for implementing and for rejecting this feature. You may be convinced that it is mistake to reject that idea and all arguments for closing were poor and unsufficient but please do not claim that there were not mentioned. Some were mentioned in a closing comment!

**Comment by matkoniecz** [comment]
> Do you have a citation to back that up? Have formal studies been done?

Do you have citation or formal studies to make an opposite claim? While it would be nice and ideal world we would use high quality studies to guide over decisions this is done very rarely if ever.

I am not sure is it a good idea to suddenly demand them from the opposite side, without providing ones supporting your viewpoint.

**Comment by matkoniecz** [comment]
> You can't really use the "clutter" or "to many icons" argument here because bench's are already being render. The bottom half of the bench with the back rest is exactly like the normal bench. Your making a major assumption about the intelligence of mappers if you believe they wont make the connection between the two in this case. I'm pretty sure 99% of people are pretty acclimated to how the normal bench icon looks at this point. So I don't see why they couldn't deduce what the new icon.

I made (without even attempt to double-blind, extremely low sample size, confounded as heck etc) some tests and typical person is unable to recognize even seemingly obvious icons.

I think that it is desirable to make map where ebing an experienced OSM mapper is not prerequisite to understand it.

For example many people were confuse by amenity=place_of_worship without icon, amenity=police icon, many shop icons, never noticed that all shop icons are purple so weird purple icons are another shops.

People also are generally confused by landuse rendering, bridge styling, tunnel styling, intermittent waterways etc.

**Comment by Adamant36** [comment]
@matkoniecz, yes decisions to a degree should be based on arguments, but none were provided for closing it, at least not one pertaining to this particular issue. It was more a general statement about detail that could literally apply to every open issue here. Although its not popularity contest, it is a community project is it not? So the opinions of the community should be considered and things shouldn't be closed based on a majority opinion or without at least testing it first or having reasoned discussion. That doesn't in any make it a popularity contest.

Once again, no reason was given that where specific to this particular issue. It was just a general statement that can apply any issue here. If this particular icon won't be recognizable, then show some evidence for it, but don't use SomeoneElse's map or some general thing about "clutter" as an excuse. Otherwise, we risk every issue and PR dissenting bickering over cartographic dogma. With nothing getting done. 

I don't think its on @eehpcm to defend his position here. Its on the person that made the general statement about clutter to say how it specifically applies in this particular case. Otherwise, anyone can come along and derail anything by using the same tactic. The responsibility is on the person with the objection to prove why their objection holds water. Not the other way around. Last time I checked there is a feedback mechanism in place for people who have an issue with something on the map to complain about it if need be to. So if its true no one will understand it, why not let it be implemented, let someone complain, then we can revise the icon to be understandable later. Ultimately though its an icon issue though, not a "should this be implemented in the first place" issue. 

I also think its a weak argument to expect everyone to prove why their imaginary people who do or don't recognize the icon is right. As I said, the icon is exactly like the original bench icon, just with a back rest added to it. Its not a new icon. We aren't reinventing the wheel. Everyone here recognizes that its a bench with a backrest, even the people that are against it, and we are all mappers. Maybe we didn't poll 2000 random people to find if they get it, but when do we? And saying people don't understand tunnel styling so they won't understand a backrest on a bench is a good what aboutism, but they are completely different and have nothing to do with each other. Plus, most people don't understand most things when they are first introduced to them, that's not an argument for or against anything though. The question is will they understand it in this case and I think they will. Since like I said, _its almost the exact same icon we are already using with a minor addition. Its not a completely different icon._

**Comment by matkoniecz** [comment]
> none were provided for closing it

You may be convinced that it is mistake to reject that idea and all arguments for closing were poor and unsufficient but please do not claim that there were not mentioned. Some were mentioned in a closing comment!

See https://github.com/gravitystorm/openstreetmap-carto/issues/3187#issuecomment-421650821

**Comment by polarbearing** [comment]
TLDR

**Comment by matkoniecz** [comment]
> It was just a general statement that can apply any issue here.

It can be aplied to most "lets show detail of XYZ using a new icon", but for example it would not apply to #3396 #3394 #3393 #3392 #3385 #3355 (from most recent ones)

**Comment by imagico** [comment]
> I didn't count Imagico's comment

As a general rule you can count me as 'boycott' in any attempt to turn design decisions into a popularity contest.  The whole idea of voting on design decisions is a transparent attempt to scuttle the argument and evidence based development process.  In a venue like this it is always easier to organize and astroturf popular support for a change than to find and present substantial arguments in support of it and to substantially consider and address critique, in particular of course if the change is a bad idea.

And to make this perfectly clear - the idea of making design decisions based on popular vote - even if including or being representative for the current mapper community - would be against the documented goals of this style.

**Comment by sommerluk** [comment]
> There is no point in our Terms of use (https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md) that maintainers votes are more important than other users opinion

There is not mentioned in CONTRIBUTING.md that there is a vote at all. In last consequence, the maintainers of this style take the decisions. 

> So the opinions of the community should be considered and things shouldn't be closed based on a majority opinion […] That doesn't in any make it a popularity contest.

The group of people participating in this discussion is around a dozen. That’s not really representative for the OSM community as a whole either.

> I don't think its on @eehpcm to defend his position here. […]  The responsibility is on the person with the objection to prove why their objection holds water.

No. If someone proposes something, he has to give arguments in favour of it.

 > @matthijsmelissen closed the issue. I think after this situation he shouldn't have the power to closing them, or at least get some warning as he showed big disrespect for another users.

No. @matthijsmelissen is a long-term maintainer and contributor of this style and is doing a great work. Honestly, I think it’s disrespect when in this discussion arguments against this change are ignored: When reading some replies here, I have the impression that the arguments against this change are indeed ignored of at least have not been understood completely and there was no real effort to understand them. Who is maintainer of this style is not object to public vote. Your behaviour with @matthijsmelissen was inappropriate.

**Comment by matthijsmelissen** [comment]
The reason of closing it quickly was that there was talk about implementing this issue, and I really want to avoid people spending time on building something that will be rejected anyway. So I think a decision was necessary. What would you have preferred?

**Comment by sommerluk** [comment]
> it was completely unexpected

It wasn’t unexpected at all. At the time @matthijsmelissen closed this issue, arguments had been given yet. (And after closing, nothing substantially new had been added.)

**Comment by kocio-pl** [comment]
The reason and action was perfectly valid IMO - it's what you think is the right solution in controversial case. Somebody will be not happy either way, but even bad decisions are better than lack of them.

My guideline in cases where more people invest their time, is to make some decision at the end of the day (as you did here), but to let people know what's going to happen. This way people have a chance to cool down a bit and have the "last minute" to add something or test if you're aware of some important details. The disappointment is expected anyway, but this way I avoid making people angry, which could lead to unnecessary hard words. I also give a warning when people use hard words already and it works like 99% without need of any action (I'm a moderator on polish OSM forum). It all boils down to the slower timing and communicating intentions IMO, especially when related to closing, locking etc.

It was unexpected for me personally, because Matthijs was absent for a long time and I would expect him to take stand in discussion and reply once or twice before making any action, so I know he is taking care of the problem.

Some other factors that I think were important here and contributed to the unfortunate clash:
- Matthijs is well known for some of people involved in this discussion, but there are people who have no opportunity to work with him before and might not see him as a gentle person
- first activity after long time is rejecting something (negative action) and there were no positive actions (like merging or openining) from him lately to balance the impression

**Comment by matthijsmelissen** [comment]
> My guideline in cases where more people invest their time, is to make some decision at the end of the day (as you did here), but to let people know what's going to happen. 

Yes, good point, I will try to do this in the future.

**Comment by Tomasz-W** [comment]
@Adamant36 I hope you will make some test renderings anyway, then discussion would be about something concrete, not about imaginations of certain users.

**Comment by Adamant36** [comment]
@Tomasz-W,  I'm still learning how to work with this kind of stuff in the code, but once I get it figured out I'll do some tests renderings to see how it looks. Maybe it will help change a few people's minds to actually see it on the map. If not though, that's fine. 

Btw, I emailed you like you requested.

**Comment by matkoniecz** [comment]
@Tomasz-W  For your information, comments like

> not about imaginations of certain users

read like an insult (what is not welcomed here).

**Comment by Tomasz-W** [comment]
If we don't have any test renderings of something, we can only imagine how would it look on map.

**Comment by kocio-pl** [comment]
My guideline when dealing with too much emotions is to not talk about people at all, only about problems. Even "you're OK" might be read as an offense, because irony might be suspected.

I'm also curious how would it look like.
